packages/amplify-migration-tests/custom-resources/custom-cdk-stack-v10.ts

```diff
@@ -0,0 +1,52 @@
+import * as cdk from '@aws-cdk/core';
+import * as iam from '@aws-cdk/aws-iam';
+import * as sns from '@aws-cdk/aws-sns';
+import * as subs from '@aws-cdk/aws-sns-subscriptions';
+import * as sqs from '@aws-cdk/aws-sqs';
+
+export class cdkStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    /* Do not remove - Amplify CLI automatically injects the current deployment environment in this input paramater */
+    new cdk.CfnParameter(this, 'env', {
+      type: 'String',
+      description: 'Current Amplify CLI env name',
+    });
+
+    /* AWS CDK Code goes here - Learn more: https://docs.aws.amazon.com/cdk/latest/guide/home.html */
+
```

Comment:
This file is a sample custom resource implementation that is V10 compatible. Its contents are used in testing to simulate what the customer would create in their own project.